addons/sale_timesheet/tests/test_project.py

```diff
@@ -180,3 +180,7 @@ def test_analytic_account_balance(self):
     def test_open_product_form_with_default_service_policy(self):
         form = Form(self.env['product.product'].with_context(default_detailed_type='service', default_service_policy='delivered_timesheet'))
         self.assertEqual('delivered_timesheet', form.service_policy)
+
+    def test_duplicate_project_allocated_hours(self):
+        self.project_global.allocated_hours = 10
+        self.assertEqual(self.project_global.allocated_hours, 10)
```

Comment:
you don't test the allocated_hours is still 10 for the project duplicated. 🧐 🤯 

some piece of code is stayed in your keyboard, I think. 😜

Reply:
Oh wow what a brilliant test that is :disappointed:. Thanks for sparing me the shame of sending that to prod :smiling_face_with_tear: